src/backend/marsha/core/tests/test_api_video.py

```diff
@@ -1284,6 +1297,18 @@ def test_api_video_create_token_user_playlist_preexists(self):
         self.assertEqual(response.status_code, 401)
         self.assertFalse(models.Video.objects.exists())
 
+    def test_api_video_create_student(self):
+        """Student users should not be able to create videos."""
+        jwt_token = AccessToken()
+        jwt_token.payload["roles"] = ["student"]
+        jwt_token.payload["permissions"] = {"can_update": False}
+        response = self.client.post(
+            "/api/videos/",
+            HTTP_AUTHORIZATION=f"Bearer {jwt_token}",
+        )
+        self.assertEqual(response.status_code, 401)
```

Comment:
401 tends to prove that it did not authenticate the user when it should have. We would expect a 403 no?

Reply:
Indeed, the ressource_id was missing in the token I created in this test.